filebeat/module/haproxy/_meta/config.yml

```diff
@@ -0,0 +1,8 @@
+- module: haproxy
+  # All logs
+  {fileset}:
+    enabled: true
+
+    # Set custom paths for the log files. If left empty,
+    # Filebeat will choose the paths depending on your OS.
+    #var.paths:
```

Comment:
Add here also a reference to `var.input`

Reply:
:+1: